libclamav/ole2_extract.c

```diff
@@ -1834,6 +2521,10 @@ cl_error_t cli_ole2_extract(const char *dirname, cli_ctx *ctx, struct uniq **fil
     hdr.xbat_start            = ole2_endian_convert_32(hdr.xbat_start);
     hdr.xbat_count            = ole2_endian_convert_32(hdr.xbat_count);
 
+    encryption_info_stream_standard_t encryption_info_stream_standard;
+    copy_encryption_info_stream_standard(&encryption_info_stream_standard, &(((const uint8_t *)phdr)[4 * (1 << hdr.log2_big_block_size)]));
```

Comment:
This line should be split into at multiple lines with length checks to make sure the buffer is big enough before we use it.